src/index.html

```diff
@@ -15,8 +15,56 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700;900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a href="./index.html">
+        <img
+          src="./images/logo.png"
+          alt="Logo"
+        />
+      </a>
+      <nav>
+        <ul>
+          <li>
+            <a class="is-active">Apple</a>
```

Comment:
```suggestion
    <header class="header">
      <a href="./index.html">
        <img
          src="./images/logo.png"
          alt="Logo"
        />
      </a>
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a class="nav__link is-active">Apple</a>
```

Add classes and use them for styling